cmake/Dependencies.cmake

```diff
@@ -1349,22 +1349,20 @@ endif()
 
 if(USE_DISTRIBUTED AND USE_TENSORPIPE)
   if(MSVC)
+    caffe2_update_option(USE_TENSORPIPE OFF)
     message(WARNING "Tensorpipe cannot be used on Windows.")
   else()
     if(USE_CUDA)
       set(TP_USE_CUDA ON CACHE BOOL "" FORCE)
       set(TP_ENABLE_CUDA_IPC ON CACHE BOOL "" FORCE)
     endif()
     set(TP_BUILD_LIBUV ON CACHE BOOL "" FORCE)
-    add_compile_options(-DTORCH_USE_LIBUV)
-    include_directories(BEFORE SYSTEM ${CMAKE_CURRENT_LIST_DIR}/../third_party/tensorpipe/third_party/libuv/include)
```

Comment:
I remember I've added this one to always build with bundled libuv rather than with system one. Why are you removing it. Can you provide some test plan?

Reply:
third-party tensorpipe package has a tensorpipe_uv target that guarantees the use of bundled libuv which is specified explicitly in torch_cpu linking of this PR.